cmd/thanos/flags.go

```diff
@@ -29,13 +29,23 @@ func regCommonServerFlags(cmd *kingpin.CmdClause) (*string, *string, func(log.Lo
 	peers := cmd.Flag("cluster.peers", "Initial peers to join the cluster. It can be either <ip:port>, or <domain:port>. A lookup resolution is done only at the startup.").Strings()
 
 	gossipInterval := cmd.Flag("cluster.gossip-interval", "Interval between sending gossip messages. By lowering this value (more frequent) gossip messages are propagated across the cluster more quickly at the expense of increased bandwidth.").
-		Default(cluster.DefaultGossipInterval.String()).Duration()
+		PlaceHolder("<gossip interval>").Duration()
 
 	pushPullInterval := cmd.Flag("cluster.pushpull-interval", "Interval for gossip state syncs. Setting this interval lower (more frequent) will increase convergence speeds across larger clusters at the expense of increased bandwidth usage.").
-		Default(cluster.DefaultPushPullInterval.String()).Duration()
+		PlaceHolder("<push-pull interval>").Duration()
```

Comment:
This change will mean that the defaults are no longer surfaced in the `--help` output. Which makes me wonder, if we should remove these flags entirely now that we have `cluster.network-type` to do the same thing (albeit with less precision).

Reply:
network-type gives no flexibility - just three fixed defaults. These 2 options were here to make it more configurable. While it is not KISS (not aware of any users that want to us it), I would leave them for now, especially that we have this already and someone might already use it. 

To make it more useful, I would stick here the Defaults from our NetworkTypes? Or at least mention where to find the default values?